client-spring/build.gradle

```diff
@@ -0,0 +1,10 @@
+
+dependencies {
+
+    implementation project(':conductor-common')
+    api project(':conductor-client')
+
+    implementation "com.netflix.eureka:eureka-client:${revEurekaClient}"
+    implementation 'org.springframework.boot:spring-boot-starter'
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
```

Comment:
Since this is already included at the root project level, this dependency is not needed in this module.